src/dict/dsl.cc

```diff
@@ -1740,12 +1736,16 @@ vector< sptr< Dictionary::Class > > makeDictionaries( vector< string > const & f
     // Make sure it's not an abbreviation file
 
     int extSize = ( uncompressedDsl ? 4 : 7 );
-    if ( fileName.size() - extSize >= 5
-         && strncasecmp( fileName.c_str() + fileName.size() - extSize - 5, "_abrv", 5 ) == 0 ) {
-      // It is, skip it
-      continue;
+
+    if ( fileName.size() - extSize >= 5 ) {
+      auto fileName_without_extension = QByteArrayView( fileName ).chopped( extSize );
+      if ( fileName_without_extension.endsWith( "_abrv" ) ) {
```

Comment:
`fileName_without_extension`   lower case

Reply:
Replaced with `strncasecmp/strnicmp -> qstrnicmp`.

Reply:
This is the whole implementation of `qstrnicmp`

https://github.com/qt/qtbase/blob/18ec9c7b62f6f1a57cd544bbd2d3e9187f8cd990/src/corelib/text/qbytearray.cpp#L343-L357

Reply:
😅 is this an improvement?

![image](https://github.com/user-attachments/assets/c22b5806-b78c-41ff-9afa-6c8527dd474c)